pkg/minikube/download/download_test.go

```diff
@@ -91,7 +91,7 @@ func testPreloadDownloadPreventsMultipleDownload(t *testing.T) {
 	group.Add(2)
 	dlCall := func() {
 		if err := Preload(constants.DefaultKubernetesVersion, constants.DefaultContainerRuntime); err != nil {
```

Comment:
@dinever thanks for picking up this task, since this is a unit test, how about instead of Really checking and donwloading preload, we simply Mock the Preload func ?

similar to this test https://github.com/medyagh/minikube/blob/cd73e832dd80a5b0d5c055a93b99a4654fca02f3/pkg/drivers/kic/oci/network_create_test.go#L26

```
var dockerInspectGetterMock = func(name string) (*RunResult, error) {
	var responseInBytes bytes.Buffer
	responseInBytes.WriteString(dockerResponse)
	response := &RunResult{Stdout: responseInBytes}

	return response, nil
}

```

Reply:
Sounds good! WIll do. Thanks!